python/ql/src/experimental/Security/BufferedRWPairUndefinedBehavior.ql

```diff
@@ -0,0 +1,23 @@
+/**
+ * @name Same parameter used as reader and writer in BufferedRWPair
+ * @description Use of same parameter as reader and writer in BufferedRWPair.
+ * @kind problem
+ * @problem.severity warning
+ * @precision medium
+ * @id python/bufferedrwpair-undefined-behavior
+ * @tags reliability
+ *       security
```

Comment:
What are the security implications of this?

Reply:
Just added some CWE references here to fix this.